football/app/models/user.rb

```diff
@@ -2,5 +2,9 @@ class User < ActiveRecord::Base
 	validates :email, format: { with: /\A([^@\s]+)@((?:[-a-z0-9]+\.)+[a-z]{2,})\z/i},
                     presence: true,
                     uniqueness: {message: "alrealy exits"}
+  validates :smartphone, numericality: {only_integer: true}
+
+  validates :name, presence: true, length: { minimum:  8}
+  validates :password, presence: true, confirmation: true,length: { minimum: 8 }
 
 end
```

Comment:
Add this line `"ensure_newline_at_eof_on_save": true,` to add new line at the end of file automatically